### ISSUE TITLE
With Xcode 10 doesn't build

### ISSUE THREAD

**Issue by enx1998** [sequelpro/sequelpro#3179]
Hi, with Xcode 10.0 (10A255) repo update at the latest commit (c0ac6ed) doesn't compile because of i386 architecture. 
Here are the errors:

> Build system information
error: The i386 architecture is deprecated. You should update your ARCHS build setting to remove the i386 architecture. (in target 'SequelProTunnelAssistant')
Build system information
error: The i386 architecture is deprecated. You should update your ARCHS build setting to remove the i386 architecture. (in target 'Sequel Pro QLGenerator')
Build system information
error: The i386 architecture is deprecated. You should update your ARCHS build setting to remove the i386 architecture. (in target 'xibLocalizationPostprocessor')
Build system information
error: The i386 architecture is deprecated. You should update your ARCHS build setting to remove the i386 architecture. (in target 'PSMTabBar')

![image](https://user-images.githubusercontent.com/20546217/46084609-5c2c3b00-c1a4-11e8-8778-4d58caa638aa.png)
Is there anything I can do to do a succesful compilation?

Thanks ENx

**Comment by unCleanCode** [comment]
I tried removing i386 from ARCHS list. But then I'm getting an error of SPMySQL.h file not found, so didn't succeed in compiling it either
Really want to get new version with dark theme and bugfixes asap. That's the last light app on my computer and it blinds me at night  :(

**Comment by khanhicetea** [comment]
I built successfully the latest commit in XCode 10 (macOS Mojave). You can follow these steps on my blog article : https://khanhicetea.com/post/build-sequel-pro-from-source-in-xcode-10/

<img width="505" alt="image" src="https://user-images.githubusercontent.com/4528223/46289132-570b2980-c5b2-11e8-9551-71d90a0a498a.png">

Hope it help

**Comment by jeffcross** [comment]
Thank you, @khanhicetea! I followed your directions and with only one minor change to the copy command at the end, I am on the latest and greatest version... with Dark Mode!

When I ran the copy command you supplied, I received an error:

```bash
$ cp build/Release/Sequel\ Pro.app /Applications/Sequel\ Pro.app
cp: build/Release/Sequel Pro.app is a directory (not copied).
```

Ah, makes sense... I simply added a `-R` `to the copy command and everything is working as expected!

```bash
$ cp -R build/Release/Sequel\ Pro.app /Applications/Sequel\ Pro.app
```

<img width="614" alt="screen shot 2018-10-01 at 8 18 04 am" src="https://user-images.githubusercontent.com/3952325/46290979-eda7fe00-c552-11e8-9cc3-6c316b17a5b0.png">

**Comment by khanhicetea** [comment]
> Ah, makes sense... I simply added a `-R` `to the copy command and everything is working as expected!
> 
> ```shell
> $ cp -R build/Release/Sequel\ Pro.app /Applications/Sequel\ Pro.app
> ```

Yeah thanks, I forgot `-R` , updated my post !

**Comment by cron13** [comment]
Good day, can you share builded app please?
i don't wanna install xcode for one app

**Comment by khanhicetea** [comment]
@cron13 I upload here https://github.com/khanhicetea/sequelpro/releases (unsigned app, you as your own risk !)

**Comment by unCleanCode** [comment]
@khanhicetea thanks a lot!

**Comment by n9yty** [comment]
Thanks!  I had spent some time on this today, managing to get a build easy enough, but it would not connect to my dev box set up in a docker container under Ubuntu. Yet from my host I can use nc to verify the connection, and the last released version of SequelPro connects just fine. I downloaded your build version and it has the same problem.  No errors, just a timeout dialog. :(

**Comment by khanhicetea** [comment]
@n9yty I think you should you `mysql` CLI client to try to connect your container first. Because I use MySQL in docker for Mac to, everything works fine !

**Comment by n9yty** [comment]
mysql client works, the old version of SequelPro works, the new build(s) do not. Don't know why. Yet I can connect to our web host over ssh, so it is a bit strange. Oh well.

**Comment by stuconnolly** [comment]
Related to #2758

**Comment by stuconnolly** [comment]
Fixed in 7d9ba0f87236cf4ed2eb0d6527311a872372c4c1
